Resources/Prototypes/Catalog/Fills/Items/firstaidkits.yml

```
@@ -60,7 +63,11 @@
       - id: ClothingMaskBreathMedical
       - id: EmergencyOxygenTankFilled
       - id: EmergencyMedipen
-      - id: SyringeInaprovaline
+        amount: 2 # Funkychem - more medipens in O2 kits
+        # Funkychem - convermol for O2 kits instead of inaprovaline
+   #   - id: SyringeInaprovaline
```

Weird spacing on this line
```suggestion
        # - id: SyringeInaprovaline
```